controllers/supervisor/image_frame_buffer.py

```diff
@@ -44,9 +44,10 @@ def update_image(self, time):
                         changed = True
                         break
                 if changed:
-                    b64_encoded = ''
+                    pixels = []
                     for py in range(yStart, yEnd):
                         index = 4 * (py * self.width + xStart)
-                        b64_encoded += base64.b64encode(bytes(self.currentImage[index:index + xLength * 4])).decode("utf-8")
-                    ret.append([xStart, yStart, xLength, yLength, b64_encoded])
+                        pixels.extend(self.currentImage[index:index + xLength * 4])
+                    ret.append([xStart, yStart, xLength, yLength, base64.standard_b64encode(bytes(pixels))])
```

Comment:
The base64 encoding is sometimes returning strange values such as:
> ...'\\x00', '\\xff', **'{'**, **'{'**, **'{'**, '\\xff', '\\x00', '\\xd1'...

Not sure to understand where does these `{` come from